<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>81</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Fruit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rBtn1">
         <property name="text">
          <string>Apple</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtn2">
         <property name="text">
          <string>Banana</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtn3">
         <property name="text">
          <string>Lemon</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Kg</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rBtn4">
         <property name="text">
          <string>One</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtn5">
         <property name="text">
          <string>Two</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtn6">
         <property name="text">
          <string>Thre</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButton2">
      <property name="text">
       <string>Sllected</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
